Engine/Graphics/Outdoor.cpp

```diff
@@ -1250,10 +1250,19 @@ bool OutdoorLocation::Load(const String &filename, int days_played,
 
     pGameLoadingUI_ProgressBar->Progress();  //прогресс загрузки
 
-    assert(sizeof(Actor) == 836);
+    // assert(sizeof(Actor) == 836);
+    assert(sizeof(Actor) == 828 + 2 * sizeof(void *));
+
     // pFilename = (char *)(836 * uNumActors);
-    memcpy(pActors.data(), pSrc + 4, uNumActors * sizeof(Actor));
-    pSrc += 4 + uNumActors * sizeof(Actor);
+    // memcpy(pActors.data(), pSrc + 4, uNumActors * sizeof(Actor));
+
+    for (int i = 0; i < uNumActors; ++i) {
```

Comment:
this is a bit fishy as going x64 we declare (at least trying to ensure) that structures can be easily changed. any change to Actor will break this unfortunately. could you please create a separate actor struct for direct fread/memcpy

Reply:
I will create Actor_MM7 and initialise Actor from there.

Reply:
Resolved.